runtime/rasdump/j9dmp.tdf

```diff
@@ -41,3 +41,4 @@ TraceEvent=Trc_dump_unwindAfterSilentDump_Event1 NoEnv Overhead=1 Level=4 Templa
 TraceAssert=Assert_dump_true noEnv Overhead=1 Level=1 Assert="(P1)"
 
 TraceExit=Trc_trcengine_criu_criuReloadXDumpAgents_Exit Overhead=1 Level=5 Template="criuReloadXDumpAgents() returns %d"
```

Comment:
We don't normally change the signature of existing tracepoints, but given this one was added only today, I think we can make an exception. The argument is of type `IDATA` so `%d` should be `%zd`.
Likewise for `Trc_trcengine_criu_criuRestoreInitializeTrace_Exit`.

Reply:
Fixed